docs/integrations/sources/hubspot.md

```diff
@@ -22,6 +22,7 @@ You can use OAuth, API key, or Private App to authenticate your HubSpot account.
 | `engagements_emails`        | `sales-email-read`                                                                                           |
 | `forms`                     | `forms`                                                                                                      |
 | `form_submissions`          | `forms`                                                                                                      |
+| `goals`                     | `crm.objects.goal_targets.read`                                                                              |
```

Comment:
Need update `crm.objects.goal_targets.read` to `crm.objects.goals.read`

Reply:
That's very sloppy of me, it is done now, thanks!